### DISCUSSION TITLE
Import saved recipes

### DISCUSSION THREAD

**Discussion by Smolders2007** [TandoorRecipes/recipes#2668] [Q&A]
My computer crashed so I have start from scratch and only have an old (December 2022) saved zip file made with the Tandoor recipes.

I started a new proxmox  server and installed Tandoor recipes in docker with portainer. When I try to import this fis zip file in the app section it failed because of a missing resource. The zip file has more than a hundred recipes with its images.

What am I doing wrong?

Regards,
Jean Smolders
jean.smolders@outlook.com

**Comment by nibblerrick** [comment]
Have you clicked "Tandoor" on the "please select app" page? I missed that and had the same error. After clicking on that everything went smooth.

**Reply by Smolders2007** [reply]
That did the trick, I thoufgt that that was a header. Would be actually nice if that choice was the default choice and highlighted.

**Reply by nibblerrick** [reply]
Glad I could help.
Yes, probably it could be highlighted better. Or the error message "please choose an option" instead of the resource-error.
